### ISSUE TITLE
Feature Request: Open a new terminal to run receipt like Makefile extension

### ISSUE THREAD

**Issue by cdfq152313** [nefrob/vscode-just#41]
Is it possible to run a receipt on a new terminal?

It would help to see the receipt log immediately after running.

Currently, I need to open the output and switch to vscode-just to see the result.

Reference: [Makefile Extension](https://devblogs.microsoft.com/cppblog/now-announcing-makefile-support-in-visual-studio-code/)
Example:
https://github.com/user-attachments/assets/28455153-565c-49b6-8b96-9f8c28679df9

**Comment by nefrob** [comment]
Yeah good idea. Would creating a new terminal per task to isolate output/logging (especially for long running tasks) or reusing the terminal as make does be preferred?

**Comment by nigels-com** [comment]
I'd probably prefer reusing the same terminal, rather than have some unbounded history to have to clean up later.

**Comment by nefrob** [comment]
@nigels-com One thing to consider here is a single terminal is will mean recipe commands get queued. For example, running a recipe that starts a docker container would block any other commands being run.

**Comment by nefrob** [comment]
Made single vs new terminal behavior configurable here: https://github.com/nefrob/vscode-just/pull/68. Will go out with the next release.
